server/src/main/java/org/elasticsearch/action/search/CanMatchPreFilterSearchPhase.java

```diff
@@ -480,6 +480,26 @@ synchronized FixedBitSet getPossibleMatches() {
         Stream<CanMatchShardResponse> getSuccessfulResults() {
             return Stream.empty();
         }
+
+        @Override
+        public void incRef() {
```

Comment:
Meh, I needed the phase to results to be ref-counted so just doing a dummy version here.

Reply:
Alternatively, should these assert or throw?